ddmrp_include_location_final/tests/test_ddmrp_include_location_final.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2025 ForgeFlow S.L. (http://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.addons.ddmrp.tests.common import TestDdmrpCommon
+
+
+class TestDdmrp(TestDdmrpCommon):
+    def test_01_incoming_quantity_from_final_location(self):
+        picking = self.pickingModel.with_user(self.user).create(
+            {
+                "picking_type_id": self.picking_type_in.id,
+                "location_id": self.supplier_location.id,
+                "location_dest_id": self.binB.id,
+                "move_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "name": "Test move",
+                            "product_id": self.productA.id,
+                            "product_uom": self.productA.uom_id.id,
+                            "product_uom_qty": 20,
+                            "location_id": self.supplier_location.id,
+                            "location_dest_id": self.binB.id,
```

Comment:
better use a location that is outside of the buffer, for more clear effect of the feature.

Reply:
done!